web/controllers/rules/issue/time_estimation.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Dwylbot.Rules.TimeEstimation do
+  @moduledoc """
+  Check errors for "in-progress and not time estimation" errors
+  """
+  def apply?(payload) do
+    payload["action"] == "labeled" && payload["label"]["name"] == "in-progress"
+  end
+
+  def check(payload) do
+    time = [
```

Comment:
@SimonLab this is a good list of our _current_ Time Estimation labels,
but could it be a [**RegEx**](https://hexdocs.pm/elixir/Regex.html) _instead_ of a List? 
e.g:
```
T\d{1,3}[mhd]
```

![image](https://user-images.githubusercontent.com/194400/27171251-d7c34e9c-51a8-11e7-87a3-142aa7fd525b.png)